packages/utxo-lib/tsconfig.json

```diff
@@ -1,7 +1,6 @@
 {
     "extends": "../../tsconfig.base.json",
     "compilerOptions": {
-        "skipLibCheck": false,
```

Comment:
without this option it doesn't pass anymore?

Reply:
Let's see after I rebase upon #11526

... But I still don't see any benefits using this option even when it pass 😅